app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,75 @@
+module TreeNode
+  module Menu
+    class Item < Node
+      set_attribute(:key) { "#{@options[:node_id_prefix]}__#{@object.feature}" }
+
+      set_attribute(:title) { _(details[:name]) }
```

Comment:
This is a carry over of how it currently works: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ops_controller/rbac_tree.rb#L120